index.js

```diff
@@ -129,6 +160,38 @@
       Object.prototype.toString.call(x).slice('[object '.length, -']'.length);
   }
 
+  //# type.parse :: String -> { namespace :: Nullable String, name :: String, version :: Number }
+  //.
+  //. Takes any String and parses it according to the [specification][3],
+  //. returning an object with a `namespace`, a `name` and a `version` field.
+  //.
+  //. ```javascript
+  //. > type.parse('my-package/List@2')
+  //. {namespace: 'my-package', name: 'List', version: 2}
+  //.
+  //. > type.parse('nonsense!')
+  //. {namespace: null, name: 'nonsense!', version: 0}
+  //.
+  //. > type.parse(Identity['@@type'])
+  //. {namespace: 'my-package', name: 'Identity', version: 1}
+  //. ```
+  /* eslint-disable key-spacing */
```

Comment:
I'd like to draw your attention here. We are breaking the `key-spacing` rule in most of the `parse` function.

* Disabling the rule altogether would open the rest of the code up for this mistake. 
* Putting the comments outside of the function was most aesthetically pleasing.

What do you think?